README.md

```diff
@@ -55,7 +55,7 @@ $ PYTHON=/usr/bin/python3.8 make
 
 ## Database schema
 
-Before running the alembic migrations make sure you  have created a database called `newdle` (or adjust the config file). Having done that, run `flask db upgrade` to upgrade the schema.
+Before running the alembic migrations make sure you  have created a database called `newdle` (or adjust the config file). Having done that, enter your virtual environment, ``source .venv/bin/activate``, and run `flask db upgrade` to upgrade the schema.
```

Comment:
I would document this in the general `## Development` section, since it makes sense to do it when doing any kind of development.

    Activate your virtualenv using ``...`` since this is required to run the various `flask` comments.

or something like this